docs/CODE_STYLE.md

```
@@ -45,12 +45,12 @@ All variable names, function names, etc., should use `snake_case` unless stated
 ### Constants
 
 **Macros**:
-- To denote their group, use a short prefix in all caps (e.g., `PREFIX_`).
-- The rest should be in all caps `SNAKE_CASE`.
+- Written in all caps `SNAKE_CASE`.
```

markdownlint-cli2 (0.17.2)

Lists should be surrounded by blank lines
null

(MD032, blanks-around-lists)